app.js

```diff
@@ -6,28 +6,26 @@ const { loadDatabase, searchDatabase } = require('./googleSpreadsheet');
 
 const app = express();
 
-const allScans = [];
+const allScans = new Map();
 
-function logScanned(uuid, matches) {
-  let allMatches = matches;
-  const now = new Date();
+function addRecentlyScanned(uuid, item, nbFound = 0) {
   // TRICK: only record uuids
   if (!/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i.test(uuid)) {
     return;
   }
-  allScans.map(item => item.status = (item.uuid === uuid) ? 'fixed' : item.status);
-  if (allMatches.length > 1) {
-    allMatches = allMatches.splice(0, 1);
-    allMatches[0].double = true;
+
+  const duplicatedItem = item;
+
```

Comment:
`duplicatedItem` is redundant with `item` we don't need 2 names for 1 thing.

Reply:
I don't know if item is a reference or a copy.
I wanted to ensure it is a copy because I am writing in it.

Reply:
I don't think this is creating any copy.

Reply:
I need to recheck that, but if that is a reference, then I need the local variable

Reply:
not really useful reply